src/StateService/StatePlugin.Persistence.cs

```diff
@@ -0,0 +1,37 @@
+using Akka.Actor;
+using Neo.IO.Caching;
+using static Neo.Ledger.Blockchain;
+using Neo.Persistence;
+using Neo.Plugins.StateService.StateStorage;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.Plugins.StateService
+{
+    public partial class StatePlugin : Plugin, IPersistencePlugin
+    {
+        void IPersistencePlugin.OnPersist(StoreView snapshot, IReadOnlyList<ApplicationExecuted> applicationExecutedList)
+        {
+            List<StateStore.Item> changes = new List<StateStore.Item>();
+            foreach (var item in snapshot.Storages.GetChangeSet().Where(p => p.State != TrackState.None))
+            {
+                changes.Add(new StateStore.Item
+                {
+                    State = item.State,
+                    Key = item.Key,
+                    Value = item.Item,
+                });
+            }
+            Store.Tell(new StateStore.StorageChanges
```

Comment:
We can't ensure that this message arrives before the next OnPersists.

Reply:
Don't messages in akka follow FIFO?  This message is ahead of next `StorageChanges` of next `OnPersist`.

Reply:
It could be https://doc.akka.io/docs/akka/current/general/actors.html#mailbox

`While using such a queue, the order of messages processed will naturally be defined by the queue’s algorithm and in general not be FIFO.`

Reply:
But here `Store` uses default mailbox instead some priority mailbox, so it should be FIFO.